docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+    mongodb:
+        extends:
+            file: docker-compose.yml
+            service: mongodb
+        mem_limit: 4g
+        labels:
+            - triton.cns.services=mongodb
+        network_mode: bridge
```

Comment:
In order to user compose file version 2 on Triton, `network_mode: bridge` is required (so `docker-compose.yml` is the Triton specific yml file).  `local-compose.yml` can be used on a swarm cluster.

https://github.com/autopilotpattern/mysql/pull/9#discussion_r59456715